.platform/routes.yaml

```diff
@@ -132,15 +132,17 @@ https://{default}/:
       # Removal of the pre-launch checklist in Custom Domains section in December 2022
       "/domains/checklist.html":  { "to": "/domains/steps.html", "code": 301 }
 
+      # CDN docs rework PR 2746
+      "/golive/steps/cloudflare.html":  { "to": "/domains/cdn/cloudflare.html", "code": 301 }
+      "/golive/steps/fastly.html":  { "to": "/domains/cdn/fastly.html", "code": 301 }
+
       # Deprecate Hugo aliases
       "/guides/general/mysql-replication.html":  { "to": "/add-services/mysql/mysql-replication.html", "code": 301 }
       "/dedicated.html":  { "to": "/dedicated-gen-2.html", "code": 301 }
       "/dedicated/(.*)":  { "to": "/dedicated-gen-2/$1", "code": 301, "regexp": true }
       "/guides/general/deploy-button.html":  { "to": "/development/deploy-button.html", "code": 301 }
       "/frameworks/deploy-button.html":  { "to": "/development/deploy-button.html", "code": 301 }
       "/development/public-ips.html":  { "to": "/development/regions.html", "code": 301 }
-      "/golive/steps/cloudflare.html":  { "to": "/domains/cdn/cloudflare.html", "code": 301 }
-      "/golive/steps/fastly.html":  { "to": "/domains/cdn/fastly.html", "code": 301 }
```

Comment:
:question: What's the purpose of this change? The redirects already existed.

Reply:
Looks like my brain had a glitch 🙈